cpp/log/cert.cc

```diff
@@ -38,6 +38,35 @@ static int X509_get_signature_nid(const X509* x) {
 
 namespace cert_trans {
 
+// Useful because util::Status clashes with Cert::Status so we have to
+// refer to it within the namespace while Cert::Status exists.
+static const util::Status& ERROR_STATUS = util::Status::UNKNOWN;
```

Comment:
Instead of using `static` for this (and the ones below), use an anonymous namespace. The linker will have some symbols and debug info to work with (which is handier in crashes, since `static` often get you `???` function names), but you get all the benefits (such as the compiler yelling at you when you've cleaned up the last call site).